sematic/db/migrations/20221202221000_resolution_versions.sql

```diff
@@ -0,0 +1,6 @@
+-- migrate:up
+ALTER TABLE resolutions ADD COLUMN client_version TEXT;
+
+-- migrate:down
+
+ALTER TABLE resolutions DROP COLUMN client_version;
```

Comment:
This needs to be commented-out, following [this model](https://github.com/sematic-ai/sematic/blob/main/sematic/db/migrations/20221004183943_remove_exception_from_runs.sql#L3-L4).